src/compile/mark/mark.ts

```diff
@@ -340,12 +367,154 @@ function getMarkGroup(model: UnitModel, opt: {fromPrefix: string} = {fromPrefix:
   ];
 }
 
+export function getLabel(model: UnitModel, data: string): LabelMark[] {
```

Comment:
Rename to getLabelMark(s)?

Reply:
This method never seem to return array with more than one item.  Do we really need an array here?  Can't the return value be LabelMark or null?